https://bugs.gentoo.org/527200

From 4e675f73a02714103fc74937f69e4159d5fbcfe7 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Fri, 31 Oct 2014 17:04:11 -0400
Subject: [PATCH] libtoolize: do not delete gnulib headers

These snippet/ headers are installed by gnulib, not libtool.  There's no
reason libtool should be trying to delete these for us (and will break
projects), so drop that logic.

People who are using gnulib can use gnulib to update/manage these.

* libtoolize.in (func_require_seen_libtool): Delete snippet/ header files
from $all_pkgaux_files.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 libtoolize.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libtoolize.in b/libtoolize.in
index d819470..d0cbfb0 100644
--- a/libtoolize.in
+++ b/libtoolize.in
@@ -1897,7 +1897,7 @@ func_require_seen_libtool ()
   # Do not remove config.guess, config.sub or install-sh, we don't
   # install them without --install, and the project may not be using
   # Automake.
-  all_pkgaux_files="compile depcomp missing ltmain.sh snippet/_Noreturn.h snippet/arg-nonnull.h snippet/c++defs.h snippet/warn-on-use.h"
+  all_pkgaux_files="compile depcomp missing ltmain.sh"
   all_pkgmacro_files="argz.m4 libtool.m4 ltdl.m4 ltoptions.m4 ltsugar.m4 ltversion.in ltversion.m4 lt~obsolete.m4"
   all_pkgltdl_files="COPYING.LIB Makefile Makefile.in Makefile.inc Makefile.am README acinclude.m4 aclocal.m4 argz_.h argz.c config.h.in config-h.in configure configure.ac configure.in libltdl/lt__alloc.h libltdl/lt__dirent.h libltdl/lt__glibc.h libltdl/lt__private.h libltdl/lt__strl.h libltdl/lt_dlloader.h libltdl/lt_error.h libltdl/lt_system.h libltdl/slist.h loaders/dld_link.c loaders/dlopen.c loaders/dyld.c loaders/load_add_on.c loaders/loadlibrary.c loaders/preopen.c loaders/shl_load.c lt__alloc.c lt__dirent.c lt__strl.c lt_dlloader.c lt_error.c ltdl.c ltdl.h ltdl.mk slist.c"
 
-- 
2.1.2

